css/properties/background-attachment.json

```
@@ -90,6 +90,7 @@
               ],
               "safari_ios": {
                 "version_added": "5",
+                "version_removed": "13",
                 "partial_implementation": true,
                 "notes": "<code>local</code> is recognized but has no effect."
```

Safari still _says_ that `fixed` is supported. This is significant for the cascade in ways that being wholly unsupported wouldn't be. So I think we need to stick with this being supported, but fix the note.

```suggestion
                "partial_implementation": true,
                "notes": "<code>fixed<code> is recognized but has no effect. See <a href='https://webkit.org/b/219324'>bug 219324</a>."
```